scripts/qa_crowbarsetup.sh

```diff
@@ -4211,6 +4211,7 @@ function onadmin_addupdaterepo
         onadmin_setup_local_zypper_repositories
         ensure_packages_installed createrepo
         createrepo -o $UPR $UPR || exit 8
+        gpg --detach-sign --armor --yes $UPR/repodata/repomd.xml
```

Comment:
Hi,

thanks for the hint, but I'm afraid I'm unable to find anything like createrepo-cloud-ptf:
    
    $ find automation -type f | xargs grep createrepo-cloud-ptf | wc -l
    0

Is this somewhere in the master branch?